src/js/views/patients/worklist/bulk-edit/bulk-edit_views.js

```diff
@@ -277,8 +286,13 @@ const BulkEditActionsBodyView = View.extend({
 
     const time = this.model.get('time');
     const isDisabled = (this.model.get('dateMulti') && !time) || !this.model.get('date');
+    const isOverdue = getIsOverdue(time, this.model.get('date'));
```

Comment:
`this.model.isOverdue()` won't work here?

Reply:
In this case, `this.model` is the bulk edit app state so we don't have access to the method.